src/scss/_checkbox.scss

```diff
@@ -60,9 +62,5 @@
 				padding-top: 12px; 
```

Comment:
Is it possible to swap px values for rem values, or o-typography functions? Forms might look funky if the typography scale has been customised.

Reply:
yeah I probably still need to do a clean up sweep of all of the scattered changes